tests/cupyx_tests/scipy_tests/special_tests/test_statistics.py

```diff
@@ -11,6 +11,9 @@ class _TestBase:
     def test_ndtr(self):
         self.check_unary_linspace0_1('ndtr')
 
+    def test_log_ndtr(self):
+        self.check_unary_linspace0_2('log_ndtr')
```

Comment:
Maybe we can use `check_unary_linspace0_1`. (Looks like that `0_1` means the range of linspace from zero to one)

Reply:
Exactly, right! Have changed it. Thanks